codegen/src/main/scala/com/soundcloud/twinagle/codegen/TwinagleServicePrinter.scala

```diff
@@ -37,17 +38,22 @@ final class TwinagleServicePrinter(
        |
        |  def server(service: $serviceName,
        |             extension: $EndpointMetadata => $Filter.TypeAgnostic = _ => $Filter.TypeAgnostic.Identity,
+       |             messageFilters: Seq[$MessageFilter] = Seq.empty,
```

Comment:
Shall we remove this change and require using `ServerBuilder` for users who want to specify `MessageFilter`s? I kinda regret adding the original `extension` parameter here, but that existed before we publicly exposed `ServerBuilder` to users. WDYT?

In any case, please move the parameter to the end of the parameter list to keep source compatibility with invocations that don't specify the argument names.

Reply:
Makes sense, can you give me an example of a service that uses `ServerBuilder`, because in all examples that I saw we use `.server` : https://github.com/soundcloud/tracks/blob/master/src/main/scala/com/soundcloud/tracks/server/App.scala#L18